deployment/modules/cloudflare/tiles/tiles.tfvars.json

```diff
@@ -1,3 +1,3 @@
 {
-  "pmtiles_file_name": "v1.pmtiles"
+  "pmtiles_file_name": ""
```

Comment:
Do we use this variable still? It just doesn't have a default anymore?

Reply:
That'll be a bug in the new flow, I'll need to pull the value from the needs variable as it now runs 5 concurrent jobs to push the tiles, then runs this job afterwards to commit the new filename